portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,19 +28,26 @@
 public class DataServlet extends HttpServlet {
 
   @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
No need to remove the doGet() function! You just have to add doPost() as a new function to the servlet. The SubtractionServlet.java file in the walkthrough contains both, as an example.

All you need to do to your doGet() function is to remove the step where you add your hardcoded messages, which you already did.

Reply:
So are the doGet() and doPost() functions going to be the same?

Reply:
Nope, they're different things, as I explained to you in the chat.